server/collectionPub/api.js

```diff
@@ -5,17 +5,45 @@ import {
 	updateCollectionPub,
 	setPrimaryCollectionPub,
 	destroyCollectionPub,
+	getCollectionPubs,
 } from './queries';
 
-const getRequestIds = (req) => {
+const getRequestIds = (req, argsFrom = req.body) => {
 	const user = req.user || {};
 	return {
 		userId: user.id,
-		collectionId: req.body.collectionId,
-		communityId: req.body.communityId,
+		collectionId: argsFrom.collectionId,
+		communityId: argsFrom.communityId,
 	};
 };
 
+app.get('/api/collectionPubs', async (req, res) => {
```

Comment:
This seems to be identical and duplicative of the `app.post()` directly below.

Reply:
Huh, not sure how that happened!